cachito/web/static/api_v1.yaml

```diff
@@ -341,6 +341,49 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/RequestUpdate'
+  "/requests/latest":
+    get:
+      operationId: cachito.web.api_v1.get_latest_request
+      summary: Get the latest request for a repo/ref
+      description: Return the latest request for a specified repo_name and ref
+      parameters:
+      - name: repo_name
+        required: true
+        in: query
+        description: The namespaced repository name (namespace/name)
```

Comment:
I know it's described here, but :stupid: me took several minutes to figure out the right format for the `repo_name`. I ended up using the `cachito/common/utils.py::get_repo_name` function to figure it out.

Wdyt about adding a note here such as "The domain and protocol/scheme must not be present in this param (e.g. https://github.com/my-org/my repo translates to my-org/my-repo)" or something along these lines.